pangea-sdk/v3/service/authn/api.go

```diff
@@ -124,6 +124,27 @@ func (a *ClientPassword) Change(ctx context.Context, input ClientPasswordChangeR
 	return request.DoPost(ctx, a.Client, "v2/client/password/change", &input, &ClientPasswordChangeResult{})
 }
 
+type ExpirePasswordRequest struct {
+	// Base request has ConfigID for multi-config projects
+	pangea.BaseRequest
+
+	ID string `json:"id"` // The identity of a user or a service.
+}
+
+// @summary Expire a user's password
+//
+// @description Expire a user's password.
+//
+// @operationId authn_post_v2_user_password_expire
+//
+// @example
+//
+//	resp, err := authncli.Client.Password.Expire(ctx, "pui_[...]")
+func (a *ClientPassword) Expire(ctx context.Context, id string) (*pangea.PangeaResponse[struct{}], error) {
```

Comment:
Are we sure we want to receive just an `id` and not a request instead? Probably in the future we'd want to expire password with email the username. If you are okey with that, it's approved.

Reply:
I'll keep this in mind going forward. Since I've already merged this API in other SDKs I'm gonna live with it for now.